### PR TITLE
[strings] waterway-stream Ru + regenerated.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

**Review by pastk — APPROVED** [review]

**Comment by pastk** [comment]
@organicmaps/translations
Please check that translation of a water Stream (Creek) is correct and is different from a River.

File: data/strings/types_strings.txt
Comment (vgrigoriu) [inline data/strings/types_strings.txt:1]: Can we also change the Romanian translation of “stream” to “pârâu”?

**Comment by RedAuburn** [comment]
> Maybe also the English version should be changed from a quite broad "Stream" to a more specific "Creek"?

Creek is a specific type of stream, and 'stream' is the standard name in EN IMO

**Comment by pastk** [comment]
> edit: huh, actually it's just the name for a stream in EN-US & EN-AU. but in the UK it is a specific type of stream: https://en.wikipedia.org/wiki/Creek

So following would be the best?
```
    en = Creek
    en-GB = Stream
```

@zyphlar WDYT?

**Review by jimcarst — COMMENTED** [review]
nl ok

**Pull request merged**